src/popup.tsx

```diff
@@ -140,12 +156,29 @@ function IndexPopup() {
           Off
         </label>
       </div>
-      <br />
       {process.env.NODE_ENV === "development" ? (
-        <button onClick={sendMessageToIsolatedWorldOnActiveTabForTest}>
-          debug mode
-        </button>
+        <div style={{ marginTop: "16px" }}>
+          Debug Mode<br />
+          <label>
+            <input
+              type="radio"
+              checked={debug}
+              onChange={(e) => setDebug(e.target.checked)}
+            />
+            On
+          </label>
+          <label>
+            <input
+              type="radio"
+              checked={!debug}
+              onChange={(e) => setDebug(!e.target.checked)}
+            />
+            Off
+          </label>
+        </div>
       ) : null}
+      <br />
+      <button onClick={start}> Start! </button>
```

Comment:
It seems that the start button is duplicated unnecessarily?👀

Reply:
As referred in https://github.com/canalun/brick-break-anywhere/commit/2614729ca72b29736c672db23768156bb82b9fc0, it's intentional.
I think now the settings menu is long enough that putting the start button both at the top and the bottom is a good choice.